npm install error with cypress 14.0.0

$ npm i
npm error code ERESOLVE
npm error ERESOLVE could not resolve
npm error
npm error While resolving: @4tw/cypress-drag-drop@2.2.5
npm error Found: cypress@14.0.0
npm error node_modules/cypress
npm error   dev cypress@"^14.0.0" from the root project
npm error
npm error Could not resolve dependency:
npm error peer cypress@"2 - 13" from @4tw/cypress-drag-drop@2.2.5
npm error node_modules/@4tw/cypress-drag-drop
npm error   dev @4tw/cypress-drag-drop@"^2.2.5" from the root project
npm error
npm error Conflicting peer dependency: cypress@13.17.0
npm error node_modules/cypress
npm error   peer cypress@"2 - 13" from @4tw/cypress-drag-drop@2.2.5
npm error   node_modules/@4tw/cypress-drag-drop
npm error     dev @4tw/cypress-drag-drop@"^2.2.5" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -8,7 +8,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [ 16 ]
+        node-version: [ 18 ]
 
     steps:
       - name: Checkout

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- Add support for Cypress 14.x.
+
 ## [2.2.5] - 2023-09-14
 
 - Add support for Cypress 13.x.

--- a/package.json
+++ b/package.json
@@ -6,8 +6,8 @@
   "types": "index.d.ts",
   "scripts": {
     "lint": "eslint --ext .js,.vue --ignore-path .gitignore .",
-    "test": "webpack-dev-server & cypress open",
-    "test:ci": "webpack-dev-server & cypress run",
+    "test": "NODE_OPTIONS=--openssl-legacy-provider webpack-dev-server & cypress open",
+    "test:ci": "NODE_OPTIONS=--openssl-legacy-provider webpack-dev-server & cypress run",
     "release": "yarn run lint && yarn run test:ci && release-it"
   },
   "prettier": "@4tw/prettier-config",
@@ -34,7 +34,7 @@
     "@release-it/keep-a-changelog": "^2.3.0",
     "@testing-library/cypress": "^7.0.2",
     "css-loader": "^3.4.0",
-    "cypress": "^13.1.0",
+    "cypress": "^14.0.0",
     "eslint": "^7.15.0",
     "prettier": "^2.2.1",
     "release-it": "^14.11.5",
@@ -48,6 +48,6 @@
     "webpack-dev-server": "^3.11.0"
   },
   "peerDependencies": {
-    "cypress": "2 - 13"
+    "cypress": "2 - 14"
   }
 }

